### ISSUE TITLE
Racer not build with last version of syntex_syntax crate.

### ISSUE THREAD

**Issue by alexb5** [phildawes/racer#316]
I try to build racer on Win7 64bit with rust v1.1.0 and get errors.

```D:\work\repos\racer>cargo update
    Updating registry `https://github.com/rust-lang/crates.io-index`
D:\work\repos\racer>cargo build --release
   Compiling winapi-build v0.1.0
   Compiling libc v0.1.8
   Compiling regex-syntax v0.2.1
   Compiling winapi v0.2.1
   Compiling unicode-xid v0.0.2
   Compiling rustc-serialize v0.3.15
   Compiling bitflags v0.3.2
   Compiling log v0.3.1
   Compiling memchr v0.1.3
   Compiling kernel32-sys v0.1.3
   Compiling aho-corasick v0.3.0
   Compiling regex v0.1.41
   Compiling env_logger v0.3.1
   Compiling term v0.2.11
   Compiling toml v0.1.21
   Compiling syntex_syntax v0.9.0
   Compiling racer v1.0.0 (file:///D:/work/repos/racer)
src/racer\ast.rs:74:39: 74:55 error: unresolved name `token::get_ident`
src/racer\ast.rs:74                     self.ident = Some(token::get_ident(ident).to_string());
                                                          ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:70:9: 100:10 note: expansion site
src/racer\ast.rs:81:44: 81:60 error: unresolved name `token::get_ident`
src/racer\ast.rs:81                                 let name = token::get_ident(name).to_string();
                                                               ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:78:21: 92:22 note: expansion site
note: in expansion of if let expansion
src/racer\ast.rs:70:9: 100:10 note: expansion site
src/racer\ast.rs:334:20: 334:36 error: unresolved name `token::get_ident`
src/racer\ast.rs:334         let name = token::get_ident(seg.identifier).to_string();
                                        ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:333:5: 342:6 note: expansion site
src/racer\ast.rs:460:34: 460:50 error: unresolved name `token::get_ident`
src/racer\ast.rs:460                 let methodname = token::get_ident(spannedident.node).to_string();
                                                      ^~~~~~~~~~~~~~~~
src/racer\ast.rs:490:33: 490:49 error: unresolved name `token::get_ident`
src/racer\ast.rs:490                 let fieldname = token::get_ident(spannedident.node).to_string();
                                                     ^~~~~~~~~~~~~~~~
src/racer\ast.rs:611:34: 611:50 error: unresolved name `token::get_ident`
src/racer\ast.rs:611                     let name = (&token::get_ident(name)).to_string();
                                                      ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:606:9: 622:10 note: expansion site
src/racer\ast.rs:635:34: 635:50 error: unresolved name `token::get_ident`
src/racer\ast.rs:635                 self.name = Some(token::get_ident(item.ident).to_string());
                                                      ^~~~~~~~~~~~~~~~
src/racer\ast.rs:671:34: 671:50 error: unresolved name `token::get_ident`
src/racer\ast.rs:671                 self.name = Some(token::get_ident(item.ident).to_string());
                                                      ^~~~~~~~~~~~~~~~
src/racer\ast.rs:714:32: 714:48 error: unresolved name `token::get_ident`
src/racer\ast.rs:714             self.name = Some((&token::get_ident(item.ident)).to_string());
                                                    ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:713:9: 715:10 note: expansion site
src/racer\ast.rs:727:32: 727:48 error: unresolved name `token::get_ident`
src/racer\ast.rs:727             self.name = Some((&token::get_ident(item.ident)).to_string());
                                                    ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:726:9: 731:10 note: expansion site
src/racer\ast.rs:743:38: 743:54 error: unresolved name `token::get_ident`
src/racer\ast.rs:743             self.generic_args.push((&token::get_ident(ty.ident)).to_string());
                                                          ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:742:9: 744:10 note: expansion site
src/racer\ast.rs:756:38: 756:54 error: unresolved name `token::get_ident`
src/racer\ast.rs:756             self.generic_args.push((&token::get_ident(ty.ident)).to_string());
                                                          ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:755:9: 757:10 note: expansion site
src/racer\ast.rs:763:22: 763:38 error: unresolved name `token::get_ident`
src/racer\ast.rs:763         let name = (&token::get_ident(_ident)).to_string();
                                          ^~~~~~~~~~~~~~~~
src/racer\ast.rs:776:27: 776:43 error: unresolved name `token::get_ident`
src/racer\ast.rs:776             self.name = (&token::get_ident(i.ident)).to_string();
                                               ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:775:9: 788:10 note: expansion site
src/racer\ast.rs:786:37: 786:53 error: unresolved name `token::get_ident`
src/racer\ast.rs:786                 self.values.push(((&token::get_ident(variant.node.name)).to_string(), point as usize));
                                                         ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:784:13: 787:14 note: expansion site
note: in expansion of if let expansion
src/racer\ast.rs:775:9: 788:10 note: expansion site
error: aborting due to 15 previous errors
Could not compile `racer`.
To learn more, run the command again with --verbose.```

Then i changed version of syntex_syntax crate in Cargo.toml 
" syntex_syntax = "*" " -> " syntex_syntax = "0.8.0" ".
All compiled without any errors.

**Comment by diaphore** [comment]
Same on Windows 8.1 64bits :
```_
$ rustc --version
rustc 1.3.0-nightly (8d432fbf1 2015-07-29)
```
```_
$ cargo build --release
 Downloading regex-syntax v0.2.1
 Downloading syntex_syntax v0.9.0
   Compiling bitflags v0.3.2
   Compiling winapi v0.2.1
   Compiling rustc-serialize v0.3.15
   Compiling winapi-build v0.1.0
   Compiling libc v0.1.8
   Compiling regex-syntax v0.2.1
   Compiling unicode-xid v0.0.2
   Compiling memchr v0.1.3
   Compiling log v0.3.1
   Compiling aho-corasick v0.3.0
   Compiling kernel32-sys v0.1.3
   Compiling regex v0.1.41
   Compiling term v0.2.11
   Compiling env_logger v0.3.1
   Compiling toml v0.1.21
   Compiling syntex_syntax v0.9.0
   Compiling racer v1.0.0 (file:///D:/source/racer)
src/racer\ast.rs:74:39: 74:55 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:74                     self.ident = Some(token::get_ident(ident).to_string());
                                                          ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:70:9: 100:10 note: expansion site
src/racer\ast.rs:74:39: 74:55 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:81:44: 81:60 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:81                                 let name = token::get_ident(name).to_string();
                                                               ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:78:21: 92:22 note: expansion site
note: in expansion of if let expansion
src/racer\ast.rs:70:9: 100:10 note: expansion site
src/racer\ast.rs:81:44: 81:60 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:334:20: 334:36 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:334         let name = token::get_ident(seg.identifier).to_string();
                                        ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:333:5: 342:6 note: expansion site
src/racer\ast.rs:334:20: 334:36 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:460:34: 460:50 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:460                 let methodname = token::get_ident(spannedident.node).to_string();
                                                      ^~~~~~~~~~~~~~~~
src/racer\ast.rs:460:34: 460:50 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:490:33: 490:49 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:490                 let fieldname = token::get_ident(spannedident.node).to_string();
                                                     ^~~~~~~~~~~~~~~~
src/racer\ast.rs:490:33: 490:49 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:611:34: 611:50 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:611                     let name = (&token::get_ident(name)).to_string();
                                                      ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:606:9: 622:10 note: expansion site
src/racer\ast.rs:611:34: 611:50 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:635:34: 635:50 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:635                 self.name = Some(token::get_ident(item.ident).to_string());
                                                      ^~~~~~~~~~~~~~~~
src/racer\ast.rs:635:34: 635:50 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:671:34: 671:50 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:671                 self.name = Some(token::get_ident(item.ident).to_string());
                                                      ^~~~~~~~~~~~~~~~
src/racer\ast.rs:671:34: 671:50 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:714:32: 714:48 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:714             self.name = Some((&token::get_ident(item.ident)).to_string());
                                                    ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:713:9: 715:10 note: expansion site
src/racer\ast.rs:714:32: 714:48 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:727:32: 727:48 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:727             self.name = Some((&token::get_ident(item.ident)).to_string());
                                                    ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:726:9: 731:10 note: expansion site
src/racer\ast.rs:727:32: 727:48 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:743:38: 743:54 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:743             self.generic_args.push((&token::get_ident(ty.ident)).to_string());
                                                          ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:742:9: 744:10 note: expansion site
src/racer\ast.rs:743:38: 743:54 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:756:38: 756:54 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:756             self.generic_args.push((&token::get_ident(ty.ident)).to_string());
                                                          ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:755:9: 757:10 note: expansion site
src/racer\ast.rs:756:38: 756:54 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:763:22: 763:38 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:763         let name = (&token::get_ident(_ident)).to_string();
                                          ^~~~~~~~~~~~~~~~
src/racer\ast.rs:763:22: 763:38 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:776:27: 776:43 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:776             self.name = (&token::get_ident(i.ident)).to_string();
                                               ^~~~~~~~~~~~~~~~
note: in expansion of if let expansion
src/racer\ast.rs:775:9: 788:10 note: expansion site
src/racer\ast.rs:776:27: 776:43 help: run `rustc --explain E0425` to see a detailed explanation
src/racer\ast.rs:786:37: 786:53 error: unresolved name `token::get_ident` [E0425]
src/racer\ast.rs:786                 self.values.push(((&token::get_ident(variant.node.name)).to_string(), point as usize));
                                                         ^~~~~~~~~~~~~~~~
note: in expansion of for loop expansion
src/racer\ast.rs:784:13: 787:14 note: expansion site
note: in expansion of if let expansion
src/racer\ast.rs:775:9: 788:10 note: expansion site
src/racer\ast.rs:786:37: 786:53 help: run `rustc --explain E0425` to see a detailed explanation
error: aborting due to 15 previous errors
Could not compile `racer`.

To learn more, run the command again with --verbose.
```

**Comment by jaxx** [comment]
#317 fixes this

**Comment by phildawes** [comment]
Have merged #317. Thanks @jaxx
